### PR TITLE
Fix preview

### PR DESCRIPTION
**Pull request by ptbrowne**
Multiple problems were fixed.

- Needed to wrap the ChartPublished inside a ChartConfiguratorState provider
- Change config that was triggering a Stardog error (:shrug:)
- Migrate config to latest version

### CODE REVIEW

File: app/pages/_preview.tsx [hunk 1]
```diff
@@ -30,7 +32,7 @@ const Page = () => {
 
 export default Page;
 
-const chartState = {
+const chartState = migrateConfiguratorState({
```
Comment (bprusinowski) [inline app/pages/_preview.tsx:35]: 👍 so it was not an immer error after all 😅

**Review by bprusinowski — APPROVED** [review]
👍

File: app/pages/_preview.tsx (hunk 1 above)
Comment (ptbrowne) [reply app/pages/_preview.tsx:35]: I added an issue for in the future check this problem #1525

**Comment by Rdataflow** [comment]
@ptbrowne big thanks 🎉

**Pull request merged**